### PR TITLE
Update airlift to 258

### PR DESCRIPTION
**Pull request by wendigo**
Changes requires as now, errors are rendered using jax-rs, rather than by the servlet container (Jetty).

See: https://eclipse-ee4j.github.io/jersey.github.io/apidocs/3.1.1/jersey/org/glassfish/jersey/server/ServerProperties.html#RESPONSE_SET_STATUS_OVER_SEND_ERROR

### CODE REVIEW

**Comment by wendigo** [comment]
@ebyhr related. Fixing

**Comment by wendigo** [comment]
@ebyhr I've disabled the generation of error pages on the Jetty side. Now we are in full control over the entities that we return to the clients.

File: core/trino-main/src/main/java/io/trino/server/ThrowableMapper.java [hunk 1]
```diff
@@ -45,20 +51,53 @@ public ThrowableMapper(ServerConfig config)
     @Override
     public Response toResponse(Throwable throwable)
     {
-        if (throwable instanceof WebApplicationException) {
-            return ((WebApplicationException) throwable).getResponse();
-        }
+        return switch (throwable) {
```
Comment (losipiuk) [inline core/trino-main/src/main/java/io/trino/server/ThrowableMapper.java:54]: This looks like step backward - can you explain inline why we need to have such an elaborate switch statemement here? Can we be less verbose?

Comment (wendigo) [reply core/trino-main/src/main/java/io/trino/server/ThrowableMapper.java:54]: We can move it to Airlift as this isn't Trino specific.

Comment (wendigo) [reply core/trino-main/src/main/java/io/trino/server/ThrowableMapper.java:54]: Previously error messages were generated in two layers: sometimes by jax-rs, sometimes by Jetty (setError was used for that). setError isn't the right approach with HTTP/2 as in the HTTP/2 there is no status line anymore. If you want to emit and error that contains a particular error line you need to use body for that.

Comment (wendigo) [reply core/trino-main/src/main/java/io/trino/server/ThrowableMapper.java:54]: After this change, the error mapping to a response happens in a single place with removes disambiguation and makes it explicit.

**Review by losipiuk — APPROVED** [review]
% comment

**Pull request merged**